daemons/gptp/common/ieee1588port.cpp

```diff
@@ -135,25 +164,37 @@ IEEE1588Port::IEEE1588Port
 
 	qualified_announce = NULL;
 
-	this->net_label = net_label;
+	this->net_label = portInit->net_label;
 
-	this->timer_factory = timer_factory;
-	this->thread_factory = thread_factory;
+	this->timer_factory = portInit->timer_factory;
+	this->thread_factory = portInit->thread_factory;
 
-	this->condition_factory = condition_factory;
-	this->lock_factory = lock_factory;
+	this->condition_factory = portInit->condition_factory;
+	this->lock_factory = portInit->lock_factory;
 
 	pdelay_count = 0;
 	sync_count = 0;
 	memset(link_delay, 0, sizeof(link_delay));
+
+	if (testMode) {
+		if (isGM) {
+			avbSyncState = 1;
```

Comment:
What does the avbSyncState values mean ? I gather that 1 means GM and 2 means not GM. Should we create a definition for these values?